src/Layers/effectLayer.ts

```diff
@@ -817,12 +817,6 @@ export abstract class EffectLayer {
      * @hidden Internal use only.
      */
     public _rebuild(): void {
-        let vb = this._vertexBuffers[VertexBuffer.PositionKind];

```

Comment:
It is not strictly required for me to remove these. I could add a dummy _rebuild-function in VertexBuffer that does nothing. If possible it could be marked as deprecated.

Reply:
No I like it, less code!